core/opendaq/device/include/opendaq/device_type_impl.h

```diff
@@ -15,7 +15,7 @@
  */
 
 #pragma once
-#include <coreobjects/component_type_impl.h>
```

Comment:
The include patch change should probably be noted in the changelog.

Reply:
The changelog already states:
- Component Type is moved from `coreobjects` to `opendaq`

I think this should suffice? @JakaMohorkoDS

Reply:
Yes, that'd be okay.